flink-tests/src/test/java/org/apache/flink/test/runtime/IPv6HostnamesITCase.java

```diff
@@ -146,15 +145,13 @@ private Inet6Address getLocalIPv6Address() {
 
                             // test whether Akka's netty can bind to the address
                             log.info("Testing whether Akka can use " + addr);
-                            try (NetUtils.Port port = NetUtils.getAvailablePort()) {
-                                final RpcService rpcService =
-                                        RpcSystem.load()
-                                                .localServiceBuilder(new Configuration())
-                                                .withBindAddress(addr.getHostAddress())
-                                                .withBindPort(port.getPort())
-                                                .createAndStart();
-                                rpcService.closeAsync().get();
-                            }
+                            final RpcService rpcService =
+                                    RpcSystem.load()
+                                            .localServiceBuilder(new Configuration())
+                                            .withBindAddress(addr.getHostAddress())
+                                            .withBindPort(0)
```

Comment:
This change doesn't seem to have any impact on the executed code (in both cases, the port doesn't seem to be set in [AkkaRpcService:138ff](https://github.com/apache/flink/blob/e9f3ec93aad7cec795c765c937ee71807f5478cf/flink-rpc/flink-rpc-akka/src/main/java/org/apache/flink/runtime/rpc/akka/AkkaRpcService.java#L138). I'm a bit surprised because I would have expected a port being specified in both cases after startup of the `AkkaRpcService`. Why is it ok to have no port set in this case? ...independently of what the test does (immediately closing the RpcService again). :thinking:  Can you shine some light here, Chesnay?

Reply:
IIRC local akka rpc services don't bind to a port at all and can only be accessed directly in the JVM.

That should mean that this test is broken since it never hits the network layer. I'll file a separate ticket for that.

Reply:
I'll revert the changes to this test and tackle it as part of FLINK-30741.